plugins/guests/fedora/cap/configure_networks.rb

```diff
@@ -17,7 +17,7 @@ def self.configure_networks(machine, networks)
           virtual = false
           interface_names = Array.new
           machine.communicate.sudo("/usr/sbin/biosdevname; echo $?") do |_, result|
-            virtual = true if result.chomp == '4'
+            virtual = true if ['4', '127'].include? result.chomp
```

Comment:
It would be great to have a comment here explaining what exit code 4 and 127 correspond to, but not a requirement.